### DISCUSSION TITLE
I can't add my school account to my device

### DISCUSSION THREAD

**Discussion by TeHaipOfTheWylfs** [memstechtips/UnattendedWinstall#139] [Q&A]
I have access to Microsoft 365 etc. through my school account. I tried to connect my school account to my personal computer, however, it pulls up an error message and doesn't let me connect

Here's the error message I got:
![image](https://github.com/user-attachments/assets/c2fb59fd-8c49-4f35-840d-8370fc70da3a)

I'd imagine that I missed some script that would cause this. Any ideas?

Thanks in advance!

**Comment by memstechtips** [comment]
Hey there and sorry for the late reply,

Since the creation of this issue I have released a new version of the XML file and a major update to [Winhance](https://winhance.net/), I recommend that you watch [this video](https://youtu.be/lrq3ph3xi50) which provides a full demo of the current version along with how to use the new version of WIMUtil and how to install the drivers etc. You now also have the ability to create an autounattend.xml file based on your selections in Winhance, so you have more control over which apps are removed and which settings are applied.

I have also created [this video](https://youtu.be/sCd1TI6AbOI) about v3.0.0 of UnattendedWinstall, I suggest that you watch it too if you're interested.

Hopefully, this will be fixed in the new version.

I'm closing this issue now, but if you need more help with the same issue, feel free to reopen it.
